app/controllers/burritos_controller.rb

```diff
@@ -1,66 +1,98 @@
 class BurritosController < ApplicationController
-  def index
-    matching_burritos = Burrito.all
-
-    @list_of_burritos = matching_burritos.order({ :created_at => :desc })
-
-    render({ :template => "burritos/index.html.erb" })
-  end
-
-  def show
-    the_id = params.fetch("path_id")
-
-    matching_burritos = Burrito.where({ :id => the_id })
-
-    @the_burrito = matching_burritos.at(0)
-
-    render({ :template => "burritos/show.html.erb" })
-  end
+  before_action :set_burrito, only: %i[ show edit update destroy ]
 
   def create
-    the_burrito = Burrito.new
-    the_burrito.name = params.fetch("query_name")
-    the_burrito.tortilla = params.fetch("query_tortilla")
-    the_burrito.image = params.fetch("query_image")
-    the_burrito.restaurant_id = params.fetch("query_restaurant_id")
-    the_burrito.owner_id = params.fetch("query_owner_id")
-    the_burrito.price = params.fetch("query_price")
-    the_burrito.ratings_count = params.fetch("query_ratings_count")
+    @burrito = Burrito.new
```

Comment:
Instead of setting each field individually, you can make a method to fetch all the params into a hash and then mass assign each column by passing the hash to the `.new` method.

```rb
@burrito = Burrito.new(burrito_params)
```

what do you think?